woodchipper/http/fastapi.py

```diff
@@ -1,80 +1,84 @@
 import uuid
-from typing import Any, Callable, Coroutine
+from typing import Callable
 
-from fastapi import FastAPI, Request, Response
+from fastapi import FastAPI, Request
+from starlette.types import ASGIApp, Receive, Scope, Send
 
 from woodchipper.context import LoggingContext, logging_ctx
 
 BLACKLISTED_HEADERS = ["authorization", "cookie"]
 
 
 class WoodchipperFastAPI:
-    def __init__(self, app: FastAPI, blacklisted_headers=BLACKLISTED_HEADERS, request_id_factory=None):
+    def __init__(
+        self,
+        app: FastAPI,
+        request_id_factory=None,
+        blacklisted_headers=BLACKLISTED_HEADERS,
+    ):
+        super().__init__()
```

Comment:
[sand] A class that doesn't inherit doesn't do anything for `__init__` so this is a no-op.

Reply:
Thanks! Accidentally left this in when switching from BaseHttpMiddleware.